pipeline/resources/update-ips-dry-run-rclone-secondary-add.out

```diff
@@ -0,0 +1,104 @@
+[1m--- sc-config.yaml[0m
+[1m+++ expected[0m
+[36m@@ -1 +1,9 @@[0m
+[31m-[][0m
+[32m+- 127.0.1.1/32[0m
+[32m+- 127.0.1.2/32[0m
+[32m+- 127.0.1.3/32[0m
+[32m+- 127.0.2.1/32[0m
+[32m+- 127.0.2.2/32[0m
+[32m+- 127.0.2.3/32[0m
+[32m+- 127.0.3.1/32[0m
+[32m+- 127.0.3.2/32[0m
+[32m+- 127.0.3.3/32[0m
+[[33mck8s[0m] Diff found for .networkPolicies.global.scApiserver.ips in sc-config.yaml (diff shows actions needed to be up to date)
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[1m--- wc-config.yaml[0m
+[1m+++ expected[0m
+[36m@@ -1 +1,9 @@[0m
+[31m-[][0m
+[32m+- 127.0.1.4/32[0m
+[32m+- 127.0.1.5/32[0m
+[32m+- 127.0.1.6/32[0m
+[32m+- 127.0.2.4/32[0m
+[32m+- 127.0.2.5/32[0m
+[32m+- 127.0.2.6/32[0m
+[32m+- 127.0.3.4/32[0m
+[32m+- 127.0.3.5/32[0m
+[32m+- 127.0.3.6/32[0m
+[[33mck8s[0m] Diff found for .networkPolicies.global.wcApiserver.ips in wc-config.yaml (diff shows actions needed to be up to date)
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_sc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_sc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_sc.yaml
+[1m--- sc-config.yaml[0m
+[1m+++ expected[0m
+[36m@@ -1 +1,9 @@[0m
+[31m-[][0m
+[32m+- 127.0.1.7/32[0m
+[32m+- 127.0.1.8/32[0m
+[32m+- 127.0.1.9/32[0m
+[32m+- 127.0.2.7/32[0m
+[32m+- 127.0.2.8/32[0m
+[32m+- 127.0.2.9/32[0m
+[32m+- 127.0.3.7/32[0m
+[32m+- 127.0.3.8/32[0m
+[32m+- 127.0.3.9/32[0m
+[[33mck8s[0m] Diff found for .networkPolicies.global.scNodes.ips in sc-config.yaml (diff shows actions needed to be up to date)
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[[34mck8s[0m] Using unencrypted kubeconfig /tmp/ck8s-apps-config/.state/kube_config_wc.yaml
+[1m--- wc-config.yaml[0m
+[1m+++ expected[0m
+[36m@@ -1 +1,9 @@[0m
+[31m-[][0m
+[32m+- 127.0.1.10/32[0m
+[32m+- 127.0.1.11/32[0m
+[32m+- 127.0.1.12/32[0m
+[32m+- 127.0.2.10/32[0m
+[32m+- 127.0.2.11/32[0m
+[32m+- 127.0.2.12/32[0m
+[32m+- 127.0.3.10/32[0m
+[32m+- 127.0.3.11/32[0m
+[32m+- 127.0.3.12/32[0m
+[[33mck8s[0m] Diff found for .networkPolicies.global.wcNodes.ips in wc-config.yaml (diff shows actions needed to be up to date)
+[1m--- sc-config.yaml[0m
+[1m+++ expected[0m
+[36m@@ -1 +1 @@[0m
+[31m-[][0m
+[32m+- 127.0.0.4/32[0m
+[[33mck8s[0m] Diff found for .networkPolicies.rcloneSync.secondaryUrl.ips in sc-config.yaml (diff shows actions needed to be up to date)
```

Comment:
I can comment that `secondaryUrl` main purpose is not really to be an endpoint for rclone.
Its intended purpose was if the used url from `s3` or `swift` didn't give all the ips/ports it needed for accessing the endpoint. 
Which happened to me one time it errored out because it couldn't access the next url in line through the specified url because of not allowing the specified ip/port that other url used so by also putting in the second url it could upload the data.
But this way works too :).

Reply:
yeah, at the beginning i was just getting this output confused and thought it made secondaryurl into a endpoint.